Bottom: 7caf2333cac8b49c79f5d9829ec060262f02356b
Top:    61d67da51d9155849464fd52ecce54d5d255a81c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 04:05:11 +0000

Refresh of disable-debug-hack

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 4f8a69b..f1a4a67 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1609,10 +1609,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 0a325f8..52182c0 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -152,7 +152,7 @@ step_do_export (HID_Attr_Val * options)
   if (filename == NULL)
     filename = "pcb-out.step";
 
-#if 0
+#if 1
   board_outline_list = object3d_from_board_outline ();
 #else
   board_outline_list = NULL;
